sound/soc/codecs/rt5682.c

```diff
@@ -1882,8 +1947,8 @@ static const struct snd_soc_dapm_route rt5682_dapm_routes[] = {
 	{"IF1_ADC Mux", "Slot 2", "IF1 23 ADC Swap Mux"},
 	{"IF1_ADC Mux", "Slot 4", "IF1 45 ADC Swap Mux"},
 	{"IF1_ADC Mux", "Slot 6", "IF1 67 ADC Swap Mux"},
-	{"IF1_ADC Mux", NULL, "I2S1"},
 	{"ADCDAT Mux", "ADCDAT1", "IF1_ADC Mux"},
+	{"AIF1TX", NULL, "I2S1"},
```

Comment:
different patch?

Reply:
The original routing path will enable the widget "I2S1" during SoundWire widget "SDWTX" is enabled. So we moved the widget "I2S1" to connect with the I2S interface "AIF1TX".